fibers.scm

```diff
@@ -24,13 +24,21 @@
   #:use-module (fibers repl)
   #:use-module (fibers timers)
   #:use-module (fibers interrupts)
+  #:use-module ((fibers posix-clocks) #:select (is-osx?))
+  #:use-module (fibers config)
   #:use-module (ice-9 threads)
   #:use-module ((ice-9 ports internal)
                 #:select (port-read-wait-fd port-write-wait-fd))
   #:use-module (ice-9 suspendable-ports)
   #:export (run-fibers spawn-fiber)
   #:re-export (sleep))
 
+(if is-osx?
+  (eval-when (eval load compile)
+  (dynamic-call "init_affinity"
+                (dynamic-link (extension-library "affinity"))))
+)
+
```

Comment:
see other comment. i would keep this out of fibers.scm.

Reply:
I have added `(init-posix-clocks)` which should be implemented by the different platform files.